cli/status.go

```diff
@@ -48,9 +52,20 @@ func statusCommand(t *core.Timetrace) *cobra.Command {
 					t.Formatter().FormatBreakTime(report),
 				},
 			}
-			out.Table([]string{"Current project", "Worked since start", "Worked today", "Breaks"}, rows, nil)
+			if format != "" {
+				format = strings.ReplaceAll(format, "{project}", project)
+				format = strings.ReplaceAll(format, "{trackedTimeCurrent}", trackedTimeCurrent)
+				format = strings.ReplaceAll(format, "{todayTime}", t.Formatter().FormatTodayTime(report))
+				format = strings.ReplaceAll(format, "{breakTime}", t.Formatter().FormatBreakTime(report))
+				format = strings.ReplaceAll(format, `\n`, "\n")
+				fmt.Printf(format)
+			} else {
+				out.Table([]string{"Current project", "Worked since start", "Worked today", "Breaks"}, rows, nil)
+			}
```

Comment:
```suggestion
				fmt.Printf(format)
				return
			}
			
			out.Table([]string{"Current project", "Worked since start", "Worked today", "Breaks"}, rows, nil)
```